ci/assign-labels.sh

```diff
@@ -49,7 +49,7 @@ label_issue() {
   fi
 
   if [[ ${DRY_RUN} != true ]]; then
-    send_gh_api_data_request repos/${NEOVIM_REPO}/issues/${issue_id} \
+    send_gh_api_data_request repos/"${NEOVIM_REPO}"/issues/"${issue_id}" \
```

Comment:
is shellcheck happy with this instead:

```suggestion
    send_gh_api_data_request "repos/${NEOVIM_REPO}/issues/${issue_id}" \
```

Reply:
Yes,, shellcheck is happy with this.  I have seen some shells not like this expansion for file paths.  As long as bash will always be used, this should be fine.